apps/server/src/services/packages/packages.service.ts

```diff
@@ -65,7 +63,15 @@ export class PackagesService {
    * @returns Promise<[Package | null]> The requested package if exist.
    */
   async findOne(idPackage: string): Promise<Package | null> {
-    const fond = await this.packageRepository.findOne({id: idPackage});
+    const fond = await this.packageRepository.findOne(
+      { id: idPackage },
+      {
+        where: {
+          id: idPackage
+        },
+        relations: ['group']
+      }
+    );
```

Comment:
let use this declaration `findOne(id?: string | number | Date | ObjectID, options?: FindOneOptions<Entity>)`.
it's more efficient so update your implementation to:
```suggestion
    const fond = await this.packageRepository.findOne( idPackage,  { relations: ['group'] } );
```

Reply:
FindOne refuses relationship, and since we are looking for by id there is no difference.

Reply:
So as long it works, we don't care about the rest ie optimization and readability. is that it?